Resources/Prototypes/Entities/Structures/Furniture/chairs.yml

```diff
@@ -237,20 +237,20 @@
     - type: FoldableVisualizer
       key: folding
   - type: MeleeWeapon
-    bluntStaminaDamageFactor: 1.75
+    bluntStaminaDamageFactor: 4.0
     attackRate: 0.9
     damage:
       types:
-        Blunt: 7
+        Blunt: 5.5
   - type: DamageWeaponOnHit
     damage:
       types:
-        Blunt: 30
+        Blunt: 3
```

Comment:
Why was this reduced from 30 to 3?

Reply:
an honest to god mistake, i needed to change StaminaDamageOnHit not DamageWeaponOnHit.